### PR TITLE
[onert] Apply ReLUGrad to ElementwiseActivationLayer

### PR DESCRIPTION
**Pull request by ragmani**
This commit applies ReLUGrad to ElementwiseActivationLayer.

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Review by glistening — APPROVED** [review]
LGTM

File: runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc [hunk 1]
```diff
@@ -62,7 +64,16 @@ void ElementwiseActivationLayer::forward(bool) { cpu::ops::ElementwiseActivation
 
 void ElementwiseActivationLayer::backward(uint32_t)
 {
-  // TODO Implement details
+  switch (_op_type)
+  {
+    case ElementwiseActivationType::kReLU:
```
Comment (jyoungyun) [inline runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc:69]: Shouldn't it be implemented differently for each type?

Comment (ragmani) [reply runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc:69]: I didn't consider data type.

Comment (ragmani) [reply runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc]: I moved setting kernels to `configure` as cpu backend.

File: runtime/onert/backend/train/ops/ElementwiseActivationLayer.h [hunk 2]
```diff
@@ -53,6 +53,9 @@ class ElementwiseActivationLayer : public ::onert::exec::train::ITrainableFuncti
   const IPortableTensor *_deriv_output;
 
   ElementwiseActivationType _op_type;
+  std::function<void(const IPortableTensor *output, const IPortableTensor *incomming,
```
Comment (jyoungyun) [inline runtime/onert/backend/train/ops/ElementwiseActivationLayer.h:56]:
```suggestion
  std::function<void(const IPortableTensor *output, const IPortableTensor *incoming,
```

File: runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc [hunk 3]
```diff
@@ -50,19 +52,37 @@ void ElementwiseActivationLayer::configure(const IPortableTensor *input, IPortab
...
+                                        getShape(incomming), getBuffer<float>(incomming),
+                                        getShape(outgoing), getBuffer<float>(outgoing));
+          };
+        }
+        else
+        {
+          throw std::runtime_error("train ElementwiseActivationLayer : This layer does not "
+                                   "suppport other ReLU except for ReLU(0-inf)");
+        }
+      }
```
Comment (jyoungyun) [inline runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc:74]:
```suggestion
      }
      else
      {
        throw std::runtime_error("train ElementwiseActivationLayer: Unsupported datatype");
      }
```

**Review by jyoungyun — APPROVED** [review]
LGTM Thank you!

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-cross-aarch64-release

**Pull request merged**